frontend/apps/reader/modules/readertypeset.lua

```diff
@@ -307,10 +307,32 @@ This stylesheet is to be used only with FB2 and FB3 documents, which are not cla
             end
             return text
         end,
-        sub_item_table = obsoleted_table,
         checked_func = function()
             return obsoleted_css[self.css] ~= nil
-        end
+        end,
+        sub_item_table = obsoleted_table,
+        separator = true,
+    })
+    table.insert(style_table, {
+        text_func = function()
+            return _("Preformatted text in TXT files") .. (G_reader_settings:has("txt_preformatted") and "" or "   ★")
```

Comment:
What does this star icon really means ?
Below, it looks like `G_reader_settings:has("txt_preformatted")` can only be `0` or nil - so it indicates only when it is set globally to 0 - and not when I reset it to nil and the global ends up being 1 ?
Feels a bit odd that it would show only when disabled - meaning the default state is `on`, which has been our historical default, but users don't know our history - and future users would feel that odd.
Do we have that kind of strange behaviour elsewhere?

Reply:
As usual, the checkbox shows the status of a setting per document, and the star - the status for new documents.

Reply:
But the star won't show if it's enabled (our default) for all document, no?